examples/autokey/autokey-setup/README.md

```diff
@@ -0,0 +1,25 @@
+# Autokey Example
+
```

Comment:
We need to update the existing [autokey tests](https://github.com/terraform-google-modules/terraform-google-kms/tree/master/test/integration/autokey_example) in order to make CI test the changes implemented in this PR.

Reply:
Do you know how to run the test locally to verify its working?

Reply:
Basically, you need to run a `terraform apply` in `test/setup`, and then `go test -v` in `test/integration/autokey_example`.

Here's the [blueprint-test doc part](https://pkg.go.dev/github.com/GoogleCloudPlatform/cloud-foundation-toolkit/infra/blueprint-test#readme-4-1-test-prerequisites) explaining with more details!

Reply:
tested in local , the test are running fine. Will check the logs